sync_check/util/chunk.go

```diff
@@ -32,51 +33,47 @@ type chunkRange struct {
 	// containB is true, containE is false, means [begin, end)
 	containB bool
 	containE bool
-	notNil   bool
 }
 
-// dumpJob is the struct for job of dump
-type dumpJob struct {
-	dbName string
-	table  string
-	column *schema.TableColumn
-	where  string
-	chunk  chunkRange
+// DumpJob is the struct for job of dump
+type DumpJob struct {
+	DbName string
+	Table  string
+	Column *model.ColumnInfo
+	Where  string
+	Chunk  chunkRange
 }
 
 // newChunkRange return a range struct
-func newChunkRange(begin, end interface{}, containB, containE bool, notNil bool) chunkRange {
+func newChunkRange(begin, end interface{}, containB, containE bool) chunkRange {
 	return chunkRange{
 		begin:    begin,
 		end:      end,
 		containE: containE,
 		containB: containB,
-		notNil:   notNil,
 	}
 }
 
-func getChunksForTable(db *sql.DB, dbname, tableName string, column *schema.TableColumn, timeRange string, chunkSize, sample int) ([]chunkRange, error) {
-	noChunks := []chunkRange{{}}
+func getChunksForTable(db *sql.DB, dbName, tableName string, column *model.ColumnInfo, where string, chunkSize, sample int) ([]chunkRange, error) {
```

Comment:
why not to pass `DumpJob ` as argument?

Reply:
checkJob will be generate after this function